src/EFCore.PG/Query/EvaluatableExpressionFilters/Internal/NpgsqlCompositeEvaluatableExpressionFilter.cs

```diff
@@ -46,15 +46,12 @@ public class NpgsqlCompositeEvaluatableExpressionFilter : RelationalEvaluatableE
         [NotNull] [ItemNotNull] readonly List<IEvaluatableExpressionFilter> _filters =
             new List<IEvaluatableExpressionFilter>
```

Comment:
A bit out of scope for this PR, but as above, you could move this up to reduce the indentation.

Reply:
Let's leave for later... I don't have very strong feelings on this in any case.